shared/src/types/vote_extensions/validator_set_update.rs

```diff
@@ -186,12 +205,19 @@ impl VotingPowersMapExt for VotingPowersMap {
                 let voting_power = voting_power.round().to_integer();
                 let voting_power: ethereum::U256 = voting_power.into();
 
-                (
-                    Token::Address(ethereum::H160(addr)),
-                    Token::Uint(voting_power),
-                )
-            })
-            .unzip()
+                let (mut fst, mut snd, mut thd) = accum;
```

Comment:
nit: could use more descriptive names here, something like
```suggestion
                let (mut hot, mut cold, mut voting_power) = accum;
```